### ISSUE TITLE
iOS and Android in app purchase documantation

### ISSUE THREAD

**Issue by Ylmzef** [PHPfox-Official/phpfox-v4-issues#2896]
There no any documantation for ios and android in app purchase.

**Comment by harrison05** [comment]
Hi @Ylmzef 

We're consider adding articles of In-app purchase feature soon. If you want to use this feature, please contact us, we can support build app with In-app purchase.

Regards.

**Comment by Ylmzef** [comment]
Is it supporting with default phpfox app?

**Comment by harrison05** [comment]
No, It doesn't support with our default app. You have to contact us to request an app with it.

Regards.

**Comment by Ylmzef** [comment]
Still waiting for documantation..

**Comment by Ylmzef** [comment]
Any update please?
